Resources/Private/Templates/Application/Form/Form.html

```diff
@@ -9,8 +9,15 @@ <h1><f:translate key="tx_ats.headline.form">General</f:translate></h1>
 
 	<f:render partial="Application/ValidationResults" />
 
-
-  	<f:form.hidden property="user" value="{user}" />
+	<f:if condition="{settings.loginPage}">
```

Comment:
The indentation is slightly off here.

Reply:
I used the beautify option for the code. Can you elaborate please what exactly is off?

Reply:
I just saw that the indentation is a mess anyways (mix of spaces and tabs). So don't worry about this for now.